docs/benchmarkdotnet.md

```diff
@@ -17,10 +17,10 @@ BenchmarkDotNet is the benchmarking tool that allows to run benchmarks for .NET,
 - [Reading the Results](#reading-the-results)
   - [Reading the Histogram](#reading-the-histogram)
   - [Reading Memory Statistics](#reading-memory-statistics)
-- [Multiple Runtimes](#multiple-runtimes)
-- [Regressions](#regressions)
 - [Profiling](#profiling)
 - [Disassembly](#disassembly)
+- [Multiple Runtimes](#multiple-runtimes)
```

Comment:
Is this change necessary? Seems like we are just moving these links few lines below.